frontend/pages/hosts/details/cards/Software/SoftwareTableConfig.tsx

```diff
@@ -244,7 +260,7 @@ const generateSoftwareTableHeaders = (deviceUser = false): IDataColumn[] => {
               type="dark"
               effect="solid"
               backgroundColor="#3e4771"
-              id={`last_used__${cellProps.row.index}`}
+              id={`last_used__${cellProps.row.original.id.toString()}`}
```

Comment:
very much a nit, but here and above: do we need the explicit `toString` call? since the value is being interpolated, the call is implicit.

Reply:
Right you are!